package.json

```diff
@@ -25,7 +25,7 @@
     "mocha-teamcity-reporter": "^4.0.0",
     "playwright": "^1.43.1",
     "sinon": "^17.0.1",
-    "ts-node": "^10.9.1",
+    "source-map-support": "^0.5.21",
```

Comment:
This matches the version that is already included within our package-lock.json.  The reason it's been available until now was that `ts-node` included it, depending on `^0.5.17`, but it was already resolved in-repo to `^0.5.21`.

Reply:
Note:  many of the `kmc` packages rely on it via the following pattern.

https://github.com/keymanapp/keyman/blob/b682f896ec95f15bd15041baf685c42ba11b3820/developer/src/kmc/package.json#L66-L71

Fortunately, it's a 'leaf' package / dependency with no dependencies of its own, so there appears to be little reason for worry about keeping this specific package around.